web/src/components/schedule/subcomponents/thursday_schedule/class_grid/messages.js

```diff
@@ -0,0 +1,24 @@
+const messages = {
+  classHeader: "Class",
+  descriptionHeader: "Description",
+  prerequisitesHeader: "Prerequisites",
+  foundationsTitle: "WCS Foundations (Level\u00A01)",
```

Comment:
Added a couple of these non-breaking spaces so we don't get weird cutoffs on medium/smaller screens

Reply:
Can you add a comment in the code explaining it too?